grc/core/blocks/embedded_python.py

```diff
@@ -88,6 +88,7 @@ def __init__(self, flow_graph, **kwargs):
         self.module_name = self.name
         self._epy_source_hash = -1
         self._epy_reload_error = None
+        self.exempt_from_id_validation = True  # Exempt epy block from blacklist id validation
```

Comment:
This could/should be a class-level variable, maybe above line 75?

Reply:
Ok. Will amend